addons/hr_holidays/models/hr_leave.py

```diff
@@ -256,6 +256,8 @@ def _default_get_request_parameters(self, values):
     request_unit_half = fields.Boolean('Half Day', compute='_compute_request_unit_half', store=True, readonly=False)
     request_unit_hours = fields.Boolean('Custom Hours', compute='_compute_request_unit_hours', store=True, readonly=False)
     request_unit_custom = fields.Boolean('Days-long custom hours', compute='_compute_request_unit_custom', store=True, readonly=False)
+    # view
+    is_hatched = fields.Boolean('Hatched', compute='_compute_is_hatched', readonly=True)
```

Comment:
```suggestion
    is_hatched = fields.Boolean('Hatched', compute='_compute_is_hatched')
```
a compute field is readonly by default

Reply:
Done :)